src/export/StructureDefinitionExporter.ts

```diff
@@ -204,7 +206,15 @@ export class StructureDefinitionExporter implements Fishable {
             const target = structDef.getReferenceName(rule.path, element);
             element.constrainType(rule, this, target);
           } else if (rule instanceof BindingRule) {
-            const vsURI = this.fishForMetadata(rule.valueSet, Type.ValueSet)?.url ?? rule.valueSet;
+            const vsURI =
+              this.fishForMetadata(rule.valueSet, Type.ValueSet)?.url.split('|')[0] ??
+              rule.valueSet;
```

Comment:
I think this will end up changing the `vsURI` that we bind to so that the ValueSet cannot have a version appended. Here is an example of how this could happen:
```
Profile: MyPatient
Parent: Patient
* identifier.type from MyVS

ValueSet: MyVS
* ^url = "http://valueset.org|4.1"
```
on this branch, that `4.1` is lost when binding, but on `master`, it is preserved.

Reply:
Addressed in `e8b6e3f`.